@here/harp-mapview/lib/MapView.ts

```diff
@@ -456,6 +456,11 @@ export interface MapViewOptions extends TextElementsRendererOptions {
      */
     enableRoadPicking?: boolean;
 
+    /**
+     * Set to `true` to allow picking of technique information associated with objects.
+     */
+    enablePickTechnique?: boolean;
```

Comment:
Very minor, but could be names:
` enableTechniquePick?: boolean;`
as other flags... or
` enableTechniquePicking?: boolean;`